dts/arm/arduino_due.dts

```diff
@@ -0,0 +1,22 @@
+/dts-v1/;
+
+#include <atmel/sam3x.dtsi>
+
+/ {
+	model = "Arduino Due with an Atmel SAM3X8E";
+	compatible = "atmel,sam3x";
```

Comment:
Devicetree v0.1-20160524 spec says, "The property value consists of a concatenated list of null terminated strings, from most speciﬁc to most general (page 14)." Therefore I think pfalcon has them in the correct order. At this point I'm going to change it to the order pfalcon listed.